hc-nested-list-behavior.html

```diff
@@ -62,6 +62,28 @@
       'item-open-changed': 'itemOpenChanged'
     },
     /**
+     * will recursively descend down the tree closing each expanded tree node.
+     * Relies upon logic being in place to prevent two sibling nodes from being open concurrently.
+     *
+     * @param {Object} [parts = this.parts] - object containing all properties of each node
+     * @param {Array.<Object>} [items = this.parts.root] - root Object to begin closing nodes from.
+     * @return {Boolean} - returns true only if it sees a node with no children or all its child
+     *                     nodes are closed.
+     */
+    collapseTree(parts = this.parts, items = this.parts.root) {
```

Comment:
parts is an object of id : children pairs, each group of children can only have up to one open item.  If you want to collapse all the most optimal way would be to Object.keys(parts) to get every set of children, then children.find for the first child that is open, close it then break out.

If you want to collapse a tree then you would track down the tree, but since children aren't by level, they are by parent, they aren't exclusively closed, which means p1 is open and p1 would be closed, but it won't guarantee that p2c1 is closed if p1c1 is open.